### PR TITLE
segemehl improvements

### PR DESCRIPTION
**Pull request by bernt-matthias**
Actually I wanted just to change the data set label and add output of unmapped reads... ended up with also adding:

- added -f (bisulfite mapping)
- added -R clip accuracy
- added -J jump size
- made use of argument
- documented still missing parameters

### CODE REVIEW

File: tools/segemehl/segemehl.xml [hunk 1]
```diff
@@ -137,47 +151,57 @@
           <param name="first_strand_query" type="data" format="fastqsanger,fastqillumina,fastq,fasta" label="Reads from first strand" />
           <param name="second_strand_query" type="data" format="fastqsanger,fastqillumina,fastq,fasta" label="Reads from second strand" />
         </repeat>
-        <param name="maxinsertsize" type="integer" value="5000" label="Maximum size of the inserts (paired end)" help="default: 5000 (-I)" />
+        <param name="maxinsertsize" argument="-I/--maxinsertsize" type="integer" value="5000" label="Maximum size of the inserts (paired end)" help="default: 5000" />
```
Comment (bgruening) [inline tools/segemehl/segemehl.xml]: The argument is meant to be just one param. In this case you could just use argument="--maxinsertsize" and remove the `name="maxinsertsize"` altogether.

Comment (bernt-matthias) [reply tools/segemehl/segemehl.xml]: don

**Comment by bgruening** [comment]
Thanks @bernt-matthias!

**Comment by jfallmann** [comment]
too late, thx matthias

**Pull request merged**